### ISSUE TITLE
Pipes don't connect flush to tanks

### ISSUE THREAD

**Issue by kholgate** [mekanism/Mekanism#7624]
### Issue description

![image](https://user-images.githubusercontent.com/46480789/192775304-6c58a7ea-28cc-4c9f-a98f-e175f276534d.png)

As per image, there's a big ol' gap between the tank and the pipe. Is this an issue to be looked at / already fixed in a later version? 

Cheers

### Minecraft version

1.18.x or earlier (No longer being developed)

### Forge version

40.1.69

### Mekanism version

Older

### Other relevant versions

Mek 10.2.5

**Comment by toddlersizedsharkcage** [comment]
The tank doesn't take up the whole block on the sides.  The pipe won't go into another block as it isn't part of that block.  While the sides function for connection, the top and bottom are the connections that were visually designed for using the tank.  

TLDR: Not practical and won't be changed
